logprep/factory_error.py

```diff
@@ -16,6 +16,13 @@ def __init__(self):
         super().__init__("There must be exactly one definition per pipeline entry.")
 
 
+class NotExactlyOneEntryInConnectorConfigurationError(InvalidConfigurationError):
+    """Raise if there is not exactly one definition per connector."""
+
+    def __init__(self):
+        super().__init__("There must be exactly one definition for input and output connectors.")
+
+
```

Comment:
Can you double check again if we actually need this error? I didn't find any location where it is used.